man/systemd.exec.xml

```diff
@@ -1373,8 +1373,13 @@
           </tgroup>
         </table>
 
-        Note that as new system calls are added to the kernel, additional system calls might be added to the groups
-        above, so the contents of the sets may change between systemd versions.</para>
+        Note, that as new system calls are added to the kernel, additional system calls might be
+        added to the groups above. Contents of the sets may also change between systemd
+        versions. In addition, the list of system calls depends on the kernel version and
+        architecture for which systemd was compiled. Use
+        <command>systemd-analyze seccomp-filter</command> to list the actual list of system calls in
```

Comment:
I'd probably add a man page ref right here, to make it easy to click through (i doubt many folks would look for it in the footer)